Source/WebCore/dom/DocumentInlines.h

```diff
@@ -51,7 +51,11 @@ inline PAL::TextEncoding Document::textEncoding() const
     return PAL::TextEncoding();
 }
 
-inline AtomString Document::encoding() const { return AtomString::fromLatin1(textEncoding().domName()); }
+inline AtomString Document::encoding() const
```

Comment:
Is this function really hot enough that it should be inlined?

Reply:
Likely not.